rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I don't mind to add them, but only deprecated things can be **Optional** and removed/moved/declaration changes needs to be **Mandatory** IMO. If you will migrate you have to change the affected code.